server/src/config.ts

```diff
@@ -46,8 +46,8 @@ const config = {
       user: env.get("LBA_SMTP_AUTH_USER").default("xxxxx").asString(),
       pass: env.get("LBA_SMTP_AUTH_PASS").default("12345").asString(),
     },
-    sendinblueWebhookApiKey: env.get("LBA_SENDINBLUE_WEBHOOK_API_KEY").default("1234").asString(),
-    sendinblueApiKey: env.get("LBA_SENDINBLUE_API_KEY").default("1234").asString(),
+    brevoWebhookApiAccessCode: env.get("LBA_SENDINBLUE_WEBHOOK_API_KEY").default("1234").asString(),
```

Comment:
```suggestion
    brevoWebhookApiAccessCode: env.get("LBA_BREVO_WEBHOOK_API_KEY").default("1234").asString(),
```

Prévoir update côté infra + 1P

Reply:
oui. tu fais bien de rappeler le 1P.